docs/dev_database/part_specific/tilegrid.rst

```diff
@@ -0,0 +1,62 @@
+=============
+tilegrid file
+=============
+
+The ``tilegrid.json`` is a list of all tiles in the device.
+Each entry contains information related to the specific tile which is used at various stages of the database generation or bitstream creation.
+The most important parts of the data are related to frame addressing within the bitstream, grid and clock region location, list of underlying sites or the type of the tile itself.
+Before diving into this section it is advised to familiarize yourself with the 7 series :doc:`bitstream format <../../architecture/bitstream_format>` chapter.
+
+tiles
+-----
+
+Each entry has the following fields:
+
+- ``baseaddr`` - a tuple of (base address, inter-frame offset)
```

Comment:
Base address is no longer a tuple, it is just the base address.  Inter-frame offset is now located in the offset field.